subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync_iso.c

```diff
@@ -699,10 +769,15 @@ static void isr_rx(void *param)
 			lll->bn_curr = 1U;
 			lll->irc_curr++;
 
+			payload_offset = (lll->latency_event * lll->bn);
```

Comment:
Ditto: Nothing to prevent an overflow here. 

This is, IMO, a general issue in the controller as values are rarely checked.